katas/content/single_qubit_gates/amplitude_change/Verification.qs

```diff
@@ -14,8 +14,10 @@ namespace Kata.Verification {
             let reference = register => AmplitudeChange(alpha, register[0]);
             let isCorrect = CheckOperationsAreEqualStrict(1, solution, reference);
             if not isCorrect {
+                // In case of an error, this value defines the precision with which complex numbers should be displayed
```

Comment:
Please remove these comments in all files, since they're not accurate and confusing (I definitely spent a moment trying to figure out where the complex numbers are here!) The precision here is not used to display complex numbers like it was used in Complex Arithmetic, all numbers displayed with precision here are real. The code here is so straightforward, I don't think we need comments to explain what precision is - the name of the function that uses them is self-descriptive

Reply:
Removed